.env.example

```diff
@@ -1,17 +1,23 @@
 ### setting up AWS SQS
-VUE_APP_AWS_SQS_ACCESS_KEY=''
-VUE_APP_AWS_SQS_SECRET_KEY=''
-VUE_APP_AWS_REGION=''
-VUE_APP_AWS_SQS_URL=''
+VITE_APP_AWS_ACCESS_KEY_ID=''
+VITE_APP_AWS_SECRET_KEY=''
+VITE_APP_AWS_REGION=''
```

Comment:
why did you remove `sqs` from these names?

Reply:
I actually hadn't updated the .env file in a while, it seemed. In the github workflow and the github env variables, this is how it is stored.